leadiv/column-policy
# [feature/issues-57] Adds integration for codeship

![Codeship badge](https://codeship.com/projects/94c4ef20-11bd-0132-a74a-029e44e44534/status?branch=feature/issues-57)
## Whats New?

Set up files for codeship to be able to run a test to lint the sass. Cleans up the sass; fixing sass warnings. Like the one mentioned in issues #58. Also adds hologram for documentation but that has not be worked out on the codeship side yet.
## How To Test?
1. Pull this branch down
2. Run bundle install (if bundler is not installed then would need to install it)
3. Then run `scss-lint . --exclude=*.css`. No errors should return.
## Update Log
